### PR TITLE
Added PolicyUpdateCommand to CommandHandler

### PR DESCRIPTION
**Pull request by vadim-kovalyov**
In this PR:
- Added new `PolicyUpdateCommand` to receive policy updates from edgehub.
- Added Command suffix for commands. I think it'll make things more clear.
- Renamed `ServiceIdentity` to `IdentityUpdate`.

### CODE REVIEW

File: mqtt/mqttd/src/broker/bootstrap/edgehub.rs [hunk 1]
```diff
@@ -150,10 +152,12 @@ where
         broker_ready.send(()).expect("ready signal");
     });
 
+    let device_id = env::var(DEVICE_ID_ENV).context(DEVICE_ID_ENV)?;
```
Comment (vadim-kovalyov) [inline mqtt/mqttd/src/broker/bootstrap/edgehub.rs:155]: @and-rewsmith, @dmolokanov, @vipeller  I think I've asked before, but I forgot... do we really need to have "device_id" in the client_id for the CommandHandler?

Comment (vadim-kovalyov) [reply mqtt/mqttd/src/broker/bootstrap/edgehub.rs:155]: I've made this tiny refactoring for proper error message, but I'm happy to remove this completely if we don't need this.

File: mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs [hunk 2]
```diff
@@ -252,14 +252,15 @@ where
...
-        match update.downcast::<Vec<ServiceIdentity>>() {
+        match update.downcast::<AuthorizerUpdate>() {
             Ok(service_identities) => {
                 debug!(
                     "service identities update received. Service identities: {:?}",
                     service_identities
                 );
 
                 self.service_identities_cache = service_identities
+                    .0
```
Comment (dylanbronson) [inline mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs:263]: this turns it from AuthorizerUpdate -> Vec<IdentityUpdate>?

**Review by dmolokanov — APPROVED** [review]
lgtm

File: mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs (hunk 2 above)
Comment (vadim-kovalyov) [reply mqtt/mqtt-edgehub/src/auth/authorization/edgehub.rs:263]: right, AuthorizerUpdate is a tuple struct, so you can access its fields like it is a tuple.

**Review by dylanbronson — APPROVED** [review]
LGTM!

**Review by dmolokanov — APPROVED** [review]
:shipit:

**Pull request merged**